client/jetpack-cloud/sections/partner-portal/assign-license-form/index.tsx

```diff
@@ -5,24 +5,61 @@ import page from 'page';
 import { ReactElement, useCallback, useState } from 'react';
 import { useDispatch } from 'react-redux';
 import FormRadio from 'calypso/components/forms/form-radio';
+import Pagination from 'calypso/components/pagination';
 import SearchCard from 'calypso/components/search-card';
+import { SITE_CARDS_PER_PAGE } from 'calypso/jetpack-cloud/sections/partner-portal/assign-license-form/constants';
 import { addQueryArgs } from 'calypso/lib/url';
 import { recordTracksEvent } from 'calypso/state/analytics/actions';
 import { errorNotice } from 'calypso/state/notices/actions';
 import useAssignLicenseMutation from 'calypso/state/partner-portal/licenses/hooks/use-assign-license-mutation';
 import './style.scss';
 
-export default function AssignLicenseForm( { sites }: any ): ReactElement {
+function setPage( pageNumber: number ): void {
+	const queryParams = { page: pageNumber };
+	const currentPath = window.location.pathname + window.location.search;
+
+	page( addQueryArgs( queryParams, currentPath ) );
+}
+
+function setSearch( search: string ): void {
+	const queryParams = { search };
+	const currentPath = window.location.pathname + window.location.search;
+
+	page( addQueryArgs( queryParams, currentPath ) );
+}
```

Comment:
Changing the search term should reset the page back to 1.

Reply:
Good catch! I've now updated so that the page number is set to 1 when doing text search